css/portal-dashboard/level-viewer.less

```diff
@@ -85,6 +85,21 @@
 
       .activityImage {
         background-color: @cc-teal-light2;
```

Comment:
Why keep this?

Reply:
I just wanted a safe fallback color, but I'm ok with taking it out.